### ISSUE TITLE
aria-invalid not implemented correctly

### ISSUE THREAD

**Issue by Squallkae** [telekom/scale#2165]
**Scale Version**
3.0.0-beta.142

**Framework and version**
vue 3.3.4

**Current Behavior**
when I use invalid in e.g. text-field or text-area, the aria-invalid attribute will be set without true. Maybe the other fields which has invalid as prop are also affected.
![image](https://github.com/telekom/scale/assets/52067610/37e055d0-317a-443e-84ba-4e8c891fd71e)
![image](https://github.com/telekom/scale/assets/52067610/d0504f28-f1fd-442b-9d96-b04ae6a0511e)

**Expected Behavior**
aria-invalid, if set, should be set with true, otherwise the Screenreader (NVDA / JAWS) does not recognize the element as invalid.
![image](https://github.com/telekom/scale/assets/52067610/0aaf3064-4cbe-4c72-a235-39f98d56cdcd)
![image](https://github.com/telekom/scale/assets/52067610/28fa2cb7-2562-4285-ad09-56ae6ab3ba23)

**Additional context**
Same is on the Scale Documentation page, so I guess not a vue specific issue.
Finding from MMS Accessibility Report.

**Comment by felix-ico** [comment]
hi @Squallkae thanks for catching this, i opened a PR to fix the affected components - the fix should be available on the next release beta.143
